source/tutorials/file-sets.md

```diff
@@ -435,49 +545,72 @@ And then create a file set from just the ones we're interested in:
       ./src
     )
   ];
+in
+
+fs.trace sourceFiles
+
+stdenv.mkDerivation {
+  name = "fileset";
+  src = fs.toSource {
+    root = ./.;
+    fileset = sourceFiles;
+  };
+  postInstall = ''
+    cp -vr $src $out
```

Comment:
```suggestion
    cp -vr . $out
```

The source will be in the local directory

Reply:
Yeah but that's another layer of indirection. With `$src` you "only" have to know that non-reserved attrs become env vars to fully grasp the whole thing. Now you'd also have to know that other bit.

Reply:
We're already using that fact throughout the entire tutorial though, starting with the first `cp -v hello.txt $out`. It should be consistent, but I also don't want those long `/nix/store/... ->` lines in the log output.